utils/prezly/api/queries.ts

```diff
@@ -10,10 +10,21 @@ export const getSlugQuery = (slug: string) => ({
     ],
 });
 
-export const getStoriesQuery = () => ({
-    $and: [
-        ...publishedAndPublic,
-    ],
-});
+export const getStoriesQuery = (newsroomId: number, categoryId?: number) => {
+    const query: any = {
+        $and: [
+            ...publishedAndPublic,
+            { 'newsroom.id': { $in: [newsroomId] } },
```

Comment:
I'd recommend always default to using newsroom UUID for filtering. Numeric IDs is something we consider as legacy. So let's not promote its usage within the "future code". 

```suggestion
            { 'newsroom.uuid': { $in: [newsroomUuid] } },
```